using System;

namespace Exception_handling
{
    internal class Program
    {
        static void Main(string[] args)
        {
            //System.DivideByZeroException ==> ədədi sıfıra bölməklə yaranan xətanı idarə edir.
            //System.NullReferenceException==>null obyektinə istinad etməklə yaranan xətanı idarə edir.
            //System.InvalidCastException==> handles the error generated by invalid typecasting.
            //System.IO.IOException==> handles the Input Output errors.
            //System.FieldAccessException==> etibarsız şəxsi və ya qorunan sahəyə girişin yaratdığı xətanı idarə edir.
            //Acar sozler => try,catch,throw,and,finally
        }
    }
}
